### ISSUE TITLE
Auto friend gacha

### ISSUE THREAD

**Issue by iagodavids** [29988122/Fate-Grand-Order_Lua#260]
Hi, i have a problem the feature auto friend gacha, it doesn't work on my xiami redmi note 5. It continues to click fare from the button

**Comment by colourspectrum** [comment]
+1. Having similar issue when it has to begin fresh summon, the mistaps are causing delays. 
Here's a short clip of how it's behaving.

[![IMAGE ALT TEXT](http://img.youtube.com/vi/w8Gxbj3PxN8/0.jpg)](http://www.youtube.com/watch?v=w8Gxbj3PxN8 "FGO - Friend Gacha Lua
")

**Comment by iagodavids** [comment]
Up
The auto friend gacha file should be like this? 
http://imgur.com/a/aBeMAaL

**Comment by MathewSachin** [comment]
@reconman @potchy Is there any specific reason for not using the scaling module in `_auto_friendgacha.lua`? It has different code for scaling right now.

https://github.com/29988122/Fate-Grand-Order_Lua/blob/e2193b7a9461b52dc641ed86bd6841c93217bbad/_auto_friendgacha.lua#L4-L18

**Comment by reconman** [comment]
@MathewSachin I don't think so. It's just that the new resolution fix method was not applied to all files. It was only applied to the gift box script 2 months ago.

The scaling method as it is right now exists since 349f8b8ac7c7ad11dafdbc3c746ef4698463d6ba. The only change to the script since then was 0f07220cf79146b40c95b748638b6bb198efc8f3 with some changed coordinates.

**Comment by MathewSachin** [comment]
Do you think using the `scaling` module here would fix this issue?

**Comment by reconman** [comment]
Probably, but I don't have a JP account and don't have an emulator set up, so I can't determine the click locations for JP.

**Comment by MathewSachin** [comment]
Well, I'm also an NA only player.
